MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -71,6 +71,10 @@ - (void)setUp {
   [MSKeychainUtil clear];
   self.sut = [MSDistribute new];
   self.settingsMock = [MSMockUserDefaults new];
+  
+  // Mock MSUserDefaults shared method to return settingsMock.
+  id userDefaultsMock = OCMClassMock([MSUserDefaults class]);
```

Comment:
Moved mocking to MSMockUserDefaults. Also, i think a little bit of refactoring should be made to MSDistributeTests, cause there are couple of methods such as testOpenUrl, testNotDeleteUpdateoken, testDeleteUpdateTokenAfterReinstall, and probably some others, that implement their own mocking of MSUserDefaults. To avoid redundancy and inconsistent use, I think they should all use this mock. So, if you agree with that, i can do this refactoring :)